OSDOCS-1790: Adding auto generated CLI output (#31403)
openshift/openshift-docs: cli_reference/openshift_cli/administrator-cli-commands.adoc

[id="cli-administrator-commands"]
= OpenShift CLI administrator command reference
include::modules/common-attributes.adoc[]
:context: cli-administrator-commands

toc::[]

> This reference provides descriptions and example commands for OpenShift CLI (`oc`) administrator commands. Information on developer commands can be found in the xref:../../cli_reference/openshift_cli/developer-cli-commands.adoc#cli-developer-commands[OpenShift CLI developer command reference].

Reviewer: I might tighten it a bit, like: 
```suggestion
This reference provides descriptions and example commands for OpenShift CLI (`oc`) administrator commands. For developer commands, see the xref:../../cli_reference/openshift_cli/developer-cli-commands.adoc#cli-developer-commands[OpenShift CLI developer command reference].
```
